modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectViewLocalServiceImpl.java

```diff
@@ -286,15 +301,67 @@ private List<ObjectViewSortColumn> _addObjectViewSortColumns(
 			});
 	}
 
-	private void _validate(
+	private void _validate(long objectViewId, long objectDefinitionId)
+		throws PortalException {
+
+		ObjectView objectView = objectViewPersistence.fetchByODI_DOV_First(
+			objectDefinitionId, true, null);
+
+		if ((objectView != null) &&
+			(objectView.getObjectViewId() != objectViewId)) {
+
+			throw new DefaultObjectViewException(
+				"There can only be one default object view");
+		}
+	}
+
+	private void _validateObjectViewColumn(
+			ObjectView objectView, List<ObjectViewColumn> objectViewColumns)
+		throws ObjectViewColumnFieldNameException {
+
+		List<ObjectField> objectFields =
+			_objectFieldPersistence.findByObjectDefinitionId(
+				objectView.getObjectDefinitionId());
```

Comment:
I think the variable "objectView" is only used once to get the ID (objectView.getObjectDefinitionId()). Could you double-check, please? 
If it is true, what do you think to pass the ID instead of the object in the parameters?

Reply:
And if you do this, I believe that it wouldn't be necessary to change the method signature and the some uses from "objectView.getObjectViewId()" to "objectView" in other places in this PR.

Reply:
Ok